src/array-interface.ts

```diff
@@ -1,7 +1,7 @@
-import {Observable} from 'rxjs';
 import {Sort} from './sort';
 import {ResourceArray} from './resource-array';
 import {Resource} from './resource';
+import {Observable} from "rxjs/internal/Observable";
```

Comment:
Just a quick suggestion for this and the other imports: the repo seems to be following the Angular standard of Single Quotes for strings. Not the biggest deal but that would make you compliant 😄